biogrid/libs/biogrid-simulator/src/lib/building/building.ts

```diff
@@ -0,0 +1,43 @@
+import { EnergyUsers } from '@biogrid/grid-simulator';
+
+/**
+ * A structure such as a building or house which uses energy to operate.
+ */
+export class Building implements EnergyUsers {
+
+  /** The amount of energy the building has. */
+  private energyCount: number;
+  /** The battery for the building. */
+  battery: unknown;
+
+  /**
+   * @param {number} energy The given amount of energy. 
```

Comment:
nit: maybe be a bit more clear in the comment for this param "Amount of energy this building has" or something similar. Also, what unit of measurement is used for the energy count?

Reply:
we decided to use joules.

Reply:
That would be helpful to have documented in the comments :)